xbmc/GUIPassword.h

```diff
@@ -60,6 +60,8 @@ class CGUIPassword : public ISettingCallback
    */
   bool CheckSettingLevelLock(const SettingLevel& level, bool enforce = false);
   bool CheckMenuLock(int iWindowID);
+  bool IsVideoUnlocked();
+  bool IsMusicUnlocked();
```

Comment:
Could this method be cost?

Reply:
No - due to  `IsMasterLockUnlocked(true)` being called